plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/webview/BrowserConnector.kt

```diff
@@ -598,6 +641,85 @@ class BrowserConnector(
         }
     }
 
+    fun parseFindingsMessages(messagesMap: Map<String, *>) {
+        try {
+            val additionalMessages = messagesMap["additionalMessages"] as? MutableList<Map<String, Any>>
```

Comment:
Why is this any?

Reply:
The typing is very complex and only the shape of additionalMessages matters. Even then I am not confident in my ability to know the shape of every possible entry here. Only the shape of the findings message matters. Introducing forced shapes would be too risky in my opinion